core-ui/src/components/Predefined/Details/HPA/HPASubcomponent.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import {
+  useGet,
+  GenericList,
+  EMPTY_TEXT_PLACEHOLDER,
+  useMicrofrontendContext,
+} from 'react-shared';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'fundamental-react';
+import { navigateToResource } from 'shared/helpers/universalLinks';
+import { Tokens } from 'shared/components/Tokens';
+import { MetricsBrief } from '../HPA/helpers';
+
+export const HPASubcomponent = props => {
+  const { t } = useTranslation();
+  const { namespaceId } = useMicrofrontendContext();
+  const { data } = useGet(
```

Comment:
To discuss: I think it would be nice to handle the error state here - as for now error won't be visible at all. We still can hide the list if there are no HPAs connected, and show nothing during loading state. `GenericList` has already `serverDataError` prop, so that shouldn't be a hassle, any opinions?